artemis/task_utils.py

```diff
@@ -25,6 +25,12 @@ def get_target(task: Task) -> str:
 
 
 def get_target_url(task: Task) -> str:
+    url = task.get_payload("url")
```

Comment:
Care to elaborate on this? I believe we store IP/domain + SSL and we were building the target out of them. Are we adding `URL` for future modules as well? (for when website is hosted with a URL prefix)

Reply:
Currently the URL task type is used for example when Bruter findings are passed to the Nuclei module - for example, the phpmyadmin bruteforcing template is executed on all found phpmyadmin URLs